spec/pQueue.js

```diff
@@ -0,0 +1,117 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import PriorityQueue from '../src/pQueue'
+
+chai.use(chaiChange)
+
+describe('PriorityQueue', () => {
+  it('exists', () => {
+    expect(PriorityQueue).to.be.a('function')
+  })
+
+  context('enqueue', (element, priority) => {
+    it('adds an element with priority (number) to the back of the queue.', () => {
+      const pQueue = new PriorityQueue
+
+      expect(() => pQueue.enqueue('foo', 1))
+        .to.alter(() => pQueue.length(), {from: 0, to: 1})
+    })
+
+    it('adds an element with priority (number) to the back of the queue.', () => {
```

Comment:
this test has the same description as the one above? Which behavior is each test testing?